### ISSUE TITLE
Backend name too long

### ISSUE THREAD

**Issue by maxgio92** [eea/eea.docker.varnish#24]
Hi,
when the dynamic backend is an AWS LB endpoint DNS name, which format is:
```
(internal-)?{lb_name}-{aws_account_id}.{aws_region}.elb.amazonaws.com 
```
the Varnish backends' names will be:
```
"server_(internal_)?{lb_name}_{aws_account_id}_{aws_region}_elb_amazonaws_com_{ip_1}"
"server_(internal_)?{lb_name}_{aws_account_id}_{aws_region}_elb_amazonaws_com_{ip_2}"
```
e.g.:

```
backend server_internal_api_lb_123456789_eu_west_2_elb_amazonaws_com_172_1_12_123 {
[...]
}

backend server_internal_api_lb_123456789_eu_west_2_elb_amazonaws_com_172_1_12_345 {
[...]
}
```

but this eventually leads to a VLC compilation error, due to the backend name's constraints:
```
Message from VCC-compiler:
Name of backend too long (max 64, is 78):
[...]
Running VCC-compiler failed, exited with 2
VCL compilation failed
```

Any idea in order to resolve this issue?
Thank you

**Comment by maxgio92** [comment]
Actually I found this workaround, by truncating the backend DNS name to 10 characters:

/varnish/src/add_backends.py:83
```
def toName(text):
    return (text.replace('.', '_').replace('-', '_').replace(':', '_'))[:10]
```
It's pretty ugly but it seems to work. Do you see any problem?

**Comment by michaelwittig** [comment]
I'm running into the same issue

**Comment by valentinab25** [comment]
solved in 4.1-6.5
